schema/run.host.event.yaml

```diff
@@ -0,0 +1,58 @@
+---
+$id: runhost
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+properties:
+  event_type:
+    type: string
+    enum:
+    - create
+    - read
+    - update
+    - delete
+  hostPayload:
```

Comment:
maybe just `payload`? (the fact that it's a host is implied by the schema name as well as the topic name)

Reply:
There is already a Payload.java that defines the runs events. I named this differently so that I could distinguish between the two types. Not sure how reconcile the two since Payloads have a different set of requirements and fields than a hostPayload.

Reply:
You can tell the code generator to use a different name for the Java class: https://github.com/joelittlejohn/jsonschema2pojo/wiki/Reference#javatype

Reply:
Done.